xarray/backends/scipy_.py

```diff
@@ -364,7 +364,7 @@ def guess_can_open(
 
         if isinstance(filename_or_obj, str | os.PathLike):
             _, ext = os.path.splitext(filename_or_obj)
-            return ext in {".nc", ".nc4", ".cdf", ".gz"}
+            return ext in {".nc", ".cdf", ".nc.gz"}
```

Comment:
updated the scipy file types per https://github.com/pydata/xarray/pull/10804#issuecomment-3354824673